media/css/firefox/whatsnew/whatsnew-142.scss

```diff
@@ -618,7 +618,8 @@ body {
 .wnp-media-frame {
     overflow: hidden;
 
-    img {
+    img,
+    video {
```

Comment:
these seemed small enough to add to shared styles, can move out into separate 143-row sass file if it looks like too many potential conflicts